requirements.dev

```diff
@@ -6,4 +6,4 @@ mypy==1.10.0
 pylint==3.2.3
 pytest==8.2.2
 pytest-cov==5.0.0
-Sphinx==7.3.7
```

Comment:
Does 7.3.7 not install for you?  What made you back it off to 7.1.2?  The 7.3.7 version installed for me, and today I can even install 8.0.2 (the most recent version).  Before backing it off I'm curious what happened when you tried it (and if maybe it was related to the first issue, with python).